python/ray/rllib/utils/timer.py

```diff
@@ -7,6 +7,35 @@
 import numpy as np
 
 
+import time
```

Comment:
These changes aren't needed any more right?

Reply:
Thought it might be useful to have some kind of logging setup, although I have no objections to removing them either

Reply:
In general, the less code added the better (and we should separate unrelated changes into another pr anyways).

Reply:
Makes sense, I see an open issue for this anyways - can talk about this more later